src/config.ts

```diff
@@ -342,29 +344,61 @@ const EXCLUDED_FEE_SOURCES = (() => {
     }
 })();
 
+const EXCHANGE_PROXY_OVERHEAD_NO_VIP = () => new BigNumber(150e3);
```

Comment:
Could this 150k be a constant by any chance? It looks like it's used a few times